components/scream/src/share/io/scream_scorpio_interface.cpp

```diff
@@ -17,7 +17,8 @@ extern "C" {
   void register_file_c2f(const char*&& filename, const int& mode);
   void set_decomp_c2f(const char*&& filename);
   void set_dof_c2f(const char*&& filename,const char*&& varname,const Int dof_len,const Int *x_dof);
-  void grid_read_data_array_c2f_real(const char*&& filename, const char*&& varname, Real *&hbuf);
+  void grid_read_data_array_c2f(const char*&& filename, const char*&& varname, Real *&hbuf);
+  void grid_read_data_array_wtime_c2f(const char*&& filename, const char*&& varname, Real *&hbuf, const Int timelevel);
```

Comment:
Keep just this one (if you really want, default time index to -1), and get rid of the other one.